hyperplonk/src/snark.rs

```diff
@@ -605,6 +611,22 @@ where
         end_timer!(pi_step);
 
         end_timer!(step);
+
+        for eval_point in points.iter() {
+            transcript.append_serializable_element(b"eval_point", eval_point)?;
+        }
+        for eval in prod_evals
```

Comment:
Similarly, it might be better to put it to PCS::batch_verify(.)

Reply:
done in [e58574f](https://github.com/EspressoSystems/hyperplonk/pull/174/commits/e58574fda5b8f45b3bbf2c6745b09214c04b26f3)